samples/forseti/forseti_conf.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2017 Google Inc.
```

Comment:
If this is intended to be a default configuration (which I think it is). Let's create a top-level directory (`conf/`) that we can put this and what will be the default rules in.